### ISSUE TITLE
Can not reproduce the results

### ISSUE THREAD

**Issue by Yead** [abhi-iyer/visual-question-answering#2]
Hi, 
Thanks for your sharing. 
I tried to run the `python train.py` with the default parameters to reproduce the results but failed, my result keeps falling down and stays in 21.00%.
![image](https://user-images.githubusercontent.com/39233259/74515921-73f79100-4f4a-11ea-88bd-f927f4351318.png)

Did I use the code in the right way?  Or could you please give me some suggestions? Thank you!

**Comment by Yead** [comment]
https://github.com/abhi-iyer/visual-question-answering/blob/36c2a661e4933c26d6dfd307be850cc5c0387cbe/train.py#L187
 In my pytorch(1.1.0),the shape of a is [B,1], but the shape of y_pred is [B]
So I changed it into `y_pred == a.long().view(-1)` then everything goes well. 
Thank you

**Comment by abhi-iyer** [comment]
How does reshaping the prediction vector change the accuracy? Not sure what this change does

**Comment by Yead** [comment]
> How does reshaping the prediction vector change the accuracy? Not sure what this change does

The shape of `y_pred == a.long()` will be [B, B], so the result of `torch.sum` will be larger than the batch size. (In the beginning, I thought the acc larger than 1 because it's a percentage....., so I thought I got an accuracy of 21%，in fact it is 2100% .）

**Comment by abhi-iyer** [comment]
please check your answer distributions, that's the best way to tell if overfitting or underfitting is occurring. the sample code for this is in "demo.ipynb".
